src/pages-conditional/what-is-ethereum.tsx

```diff
@@ -524,7 +524,7 @@ const WhatIsEthereumPage = ({
                       <Tooltip
                         content={tooltipContent({
                           apiUrl:
-                            "https://www.stateofthedapps.com/stats/platform/ethereum#new",
+                            "#",
```

Comment:
A link for this looks kind of odd. Could we modify the `tooltipContent` function to only render a Link if an `apiUrl` is provided?

Reply:
Ignore this @MahendraBishnoi29 pls 🙏🏼 

As you suggested, lets
- update the resource here to be https://dappradar.com/rankings/protocol/ethereum
- update the metric to be `3882`

Thanks!

Reply:
metric 🤔 @pettinarip